### PR TITLE
Warnings cleanup

### PR DESCRIPTION
**Pull request by endJunction**
Warnings gathered around from different compilers mainly clang and msvc.

### CODE REVIEW

**Comment by TomFischer** [comment]
Looks good. :fast_forward:

File: InSituLib/VtkMappedMeshSource.h [hunk 1]
```diff
@@ -84,9 +84,10 @@ class VtkMappedMeshSource : public vtkUnstructuredGridAlgorithm
 	/// \param properties MeshLib::Properties object
 	/// \param prop_name The name of the property vector to be mapped from
 	/// vtk-mesh to ogs-mesh
-	template<typename T> bool addProperty(vtkUnstructuredGrid &output,
-	                                      MeshLib::Properties const& properties,
-	                                      std::string const& prop_name) const
+	template <typename T>
+	bool addProperty(vtkUnstructuredGrid& /*output*/,
```
Comment (bilke) [inline InSituLib/VtkMappedMeshSource.h]: I guess the unused parameter can be removed.

Comment (endJunction) [inline InSituLib/VtkMappedMeshSource.h]: Removed; but there is more in `VtkMappedMeshSource.cpp` on `output` variable, which I didn't touch. Guess it is for another PR...

**Comment by bilke** [comment]
:+1:

**Pull request merged**